modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -27,96 +31,183 @@ The repository must be accessible from the hub cluster and be defined as a sourc
 
 .Procedure
 
-. Create a `ConfigMap` resource that describes the NICs for a group of hosts. For example:
+. Create three `ConfigMap` CRs that contain the group and site configuration. 
++
+.. Create  a `ConfigMap` CR named `group-hardware-types-configmap` to hold the hardware-specific configuration. For example:
 +
 [source,yaml]
 ----
+---
```

Comment:
Just out of curiosity - are these three dashes in this yaml and the two following yaml files correct or necessary? They show up a bit weird in the rendered doc, but if they're needed then it is what it is.

Reply:
they're not needed so I've removed them. Thanks @skopacz1